### DISCUSSION TITLE
Manual Edit Table Layout Column Sizes

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#3102] [Feature Requests]
Right-click the Table Layout's column headers to get a menu for:

```
* Resize Column (40px)
* Reset Column Width
* Hide Column
---
* Left Align
* Center Align
* Right Align
---
* Sort Ascending
* Sort Descending
* Sort Randomized
```

**Set Width**: Opens dialog with input for specific pixel width
**Reset Width**: Resets to whatever "smart" default was used
**Hide Column**: Alias for layout options

Maybe an option to Fill Width? Could allow a single column to fill the remaining space.

**Comment by rijkvanzanten** [comment]
This has since been implemented!
